src/main/java/org/railwaystations/api/resources/StationsResource.java

```diff
@@ -21,6 +26,18 @@
     private static final String LAT = "lat";
     private static final String LON = "lon";
     private static final String ID = "id";
+    
+    private static final String DEFAULT_LICENSE_LANGUAGE = "en";
+    private static final String DEFAULT_LICENSE_URL = "https://creativecommons.org/licenses/";
```

Comment:
I'm not sure, if we should have a default license URL. At the moment all licenses are CC licenses, but who knows, if this stays this way. 
I would return nothing (e.g. null) if we don't have a license or don't know the URL of it.